docs-parts/intro/Releases_lang1.rst

```diff
@@ -1,3 +1,35 @@
+3.4.0 -- December 9, 2020
+-------------------------
+* Minor: Add dj.config to be compatible with dj-python and removed dj.set (#186) #188
+* Minor: Add UUID DataJoint datatype (#180) PR #194
+* Minor: Add file external storage (#143) PR #197
+* Minor: Add S3 external storage (#88) PR #207
+* Minor: Improve dependency version compatibility handling (#228) PR #285
+* Minor: Add unique and nullable options for foreign keys (#110) PR #303
+* Minor: Add non-interactive option for dj.new (#69) #317
+* Bugfix: Handle empty password (#250) PR #279, #292
+* Bugfix: Disable GUI password if running headless (#278) PR #280, #292
+* Bugfix: Fix order to dj.kill output (#229) PR #248, #292
```

Comment:
not so much 'fix' as 'add order_by option'; also added kill_quick.. nit

Reply:
can defer until pre-release cleanup / docs, or skip

Reply:
Thanks for reviewing @ixcat. I have updated the wording now but purposely not included `dj.killQuick` yet since it is uncommitted to `master` and `external-storage` branches. Was planning on handling it once this is in `master`.

Reply:
makes sense - resolving feedback